scripts/utils/audio-utils.js

```diff
@@ -91,23 +91,32 @@ export const isSceneAudio = (id) => {
 export const fadeAudioInAndOut = (oldPlayer, newPlayer, resetCurrentTime) => {
```

Comment:
Could we have a short function that does the fade check?

```suggestion
export const playerIsFading = (player) => player.volume > 0 && player.volume < 1;

export const fadeAudioInAndOut = (oldPlayer, newPlayer, resetCurrentTime) => {
```

Reply:
Yes, added!